File: Q6/Q6.js
let rows = parseInt(prompt("Enter the number of rows:"));
for (let i = 1; i <= rows; i++) 
{
    let letter = 'A';
    for (let j = 1; j <= i; j++) 
    {
        process.stdout.write(letter + " ");
        letter = String.fromCharCode(letter.charCodeAt(0) + 1);
    }
    console.log();
}

File: Q6/Q6.go
package main

import (
	"fmt"
)

func main() {
	var rows int

	fmt.Print("Enter the number of rows: ")
	fmt.Scan(&rows)

	for i := 1; i <= rows; i++ {
		letter := 'A'
		for j := 1; j <= i; j++ {
			fmt.Printf("%c ", letter)
			letter++
		}
		fmt.Println()
	}
}
